docs/getting_started/prepare_environment.md

```diff
@@ -1,61 +1,99 @@
 # Tutorial: Prepare your Environment
 
-Before starting this tutorial, you will need to setup your development machine.  These instructions were tested on an x86_64 based machine using Ubuntu 20.04.
+Before starting this tutorial, you will need to setup your development machine.  These instructions were tested on an x86_64 based machine running Ubuntu 22.04 and a machine running Azure Linux 2.0.
```

Comment:
AZL 2.0?

Reply:
I've not personally checked them on azl3. I guess I could just leave it as azure linux without a specific version.